lib/BackgroundJob/Tasks/CreateClustersTask.php

```diff
@@ -98,6 +98,8 @@ public function execute(FaceRecognitionContext $context) {
 			$this->createClusterIfNeeded($user);
 		}
 
+		$this->config->setAppValue('facerecognition', 'recreate-clusters', 'false');
```

Comment:
There is a problem ..
When it is executed for a single user, the recreation is disabled for all. So, this should should be a user settings? :confused: